tools/oesign/main.c

```diff
@@ -523,7 +524,17 @@ static const char _usage[] =
     "        -----BEGIN RSA PRIVATE KEY-----\n"
     "\n"
     "    The resulting image is written to <EnclaveImage>.signed.so.\n"
-    "\n";
+    "\n"
+    "	   		---- OR-----\n"
```

Comment:
This is an awkward way to express a multifunction CLI tool … we should probably refactor the parameter handling so that it is consolidated across the different sub-functions. This is especially true because as part of supported digest signing for #525, we will also need to add new commands to generate the digest and to inject a signed digest into an enclave sigstruct. To accommodate that I think we will need to standardize the usage format to take an explicit command:

```
Usage: oesign { sign | hash | info | verify } … <op specific parameters>
```